applets/notification_area/status-notifier/sn-item-v0.c

```diff
@@ -847,6 +849,19 @@ new_status_cb (SnItemV0 *v0,
   queue_update (v0);
 }
 
+static void
+new_label_cb (SnItemV0 *v0,
+              GVariant *parameters)
+{
+  char *label = NULL;
+
+  g_free (v0->label);
+  g_variant_get (parameters, "(ss)", &label, NULL);
```

Comment:
This returns newly allocated string! You have memory leak here...

Reply:
PR #1204